### DISCUSSION TITLE
Existe alguma limitação para criação da funcionalidade de exportar formulário?

### DISCUSSION THREAD

**Discussion by matheusassed** [fluiggers/fluig-vscode-extension#23] [General]
A extensão tem sido muito útil para mim, mas ainda tenho o processo de ter que ir no Eclipse para exportar as alterações no formulário para o servidor. É algo que está no radar para desenvolvimento, ou há alguma limitação do Fluig com relação a isso?

**Comment by brunogasparetto** [comment]
Tenho vontade de implementar exportação de formulários, mas faltou tempo para estudar mais como funciona.

Aparentemente precisa zipar toda a pasta e enviar para o WS correto do Fluig, informando se haverá troca de versão e outros detalhes.

Devido a ter que estudar as opções dos WS e ainda desenvolver a tela de exportação acabei não prosseguindo com essa funcionalidade, ainda.

**Comment by brunogasparetto** [comment]
Estou fazendo uns testes na exportação de formulário.

Se quiser dá pra baixar o projeto no meu fork https://github.com/brunogasparetto/fluig-vscode-extension/tree/exportForm e testar se tá funcionando bem.

Não pretendo gastar muito tempo criando tela "bonita" de exportação e/ou de procura da pasta no servidor ao criar (no momento precisa colocar o id da pasta onde o formulário será criado).

**Comment by brunogasparetto** [comment]
A versão 1.20.0 agora possuí exportação de formulários.

Porém não foi implementada a opção de indicar um campo descritor ao card, porque para isso precisa analisar o HTML e exibir uma lista de todos os campos do formulário.

Se for algo muito necessário podemos estudar a possibilidade num futuro próximo (aí só abrir issue solicitando).
